ui/blocks/utils.ts

```diff
@@ -0,0 +1,12 @@
+import getValueWithUnit from 'lib/getValueWithUnit';
+
+export const getBaseFeeValue = (baseFee: string | null) => {
+  if (!baseFee) {
+    return null;
+  }
+  const valGwei = getValueWithUnit(baseFee, 'gwei');
+  if (valGwei.isGreaterThanOrEqualTo(0.0001)) {
+    return valGwei.toFormat(4) + ' Gwei';
+  }
+  return getValueWithUnit(baseFee, 'wei').toFormat() + ' wei';
```

Comment:
We should add support for custom WEI units here as well. Please refer to `lib/units.ts`